### ISSUE TITLE
Watermaker - Blown High Pressure Hose

### ISSUE THREAD

**Issue by pcbaston** [CanberraOceanRacingClub/namadgi3#695]
Michelle reported that watermaker blew a hose and dumped seawater into the bilges. She had it inspected by Michael from SeaPower during a visit to marina. Reported:

"Michael from Seapower came today to inspect the watermaker. He reattached the blown hose only to have it blow again when the motor was started. He said it must be a blocked membranes and that is going to take time to get parts etc. I told him the boat isn't booked from the 5 April to 14 April so he should aim for earlier during that period to fix it."

![20230318_182257](https://user-images.githubusercontent.com/120452016/227804425-2fc047d2-8491-4550-8f89-f8b090f7f73d.jpg)

Membranes may be blocked from use in dirty water.

**Comment by pcbaston** [comment]
Seapower has been engaged to rectify issue during the vacant NZ8 booking window.

**Comment by pcbaston** [comment]
Peter Wain advised me:
"I wondered whether there is any info on why the watermaker would have developed "blocked membranes", as it was described in an email from Michelle's email. It would be good to find out so we can prevent it from happening again. It might just be old. Also it might be worth replacing the carbon filter, which is used to remove any chlorine from the water used for flushing, and I doubt whether it has been replaced for a long time."

I forwarded this info to Mike at Seapower (Opua) 11Apr23.
